apps/importer/package.json

```diff
@@ -0,0 +1,14 @@
+{
+  "name": "importer",
+  "module": "index.ts",
+  "type": "module",
+  "scripts": {
+    "dev": "bun run src/index.ts "
```

Comment:
for testing purposes, it might be good to add a script here to create a round using your `index.ts` file